content/blog/2017-04-10-promehteus-20-sneak-peak.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Sneak Peak of Prometheus 2.0
+created_at: 2017-04-10
+kind: article
+author_name: Fabian Reinartz
+---
+
+In July 2016 Prometheus reached a big milestone: its 1.0 release. Since then plenty of new features were added such as new service discovery integrations and our experimental remote APIs.  
```

Comment:
That part sounds cumbersome either way. What about

"In July 2016 Prometheus reached a big milestone with its 1.0 release. Since then, plenty of new features like new service discovery integrations and our experimental remote APIs have been added."